string nextMode
---
string data
bool success 
string message